common/src/main/java/org/apache/falcon/lifecycle/FeedLifecycleStage.java

```diff
@@ -22,7 +22,8 @@
  */
 public enum FeedLifecycleStage {
```

Comment:
This class needs to be re-visited. A common set of policies for all lifecycles and a default policy across stages does not make sense to me. May be have a base class and child classes.